examples/complete/README.md

```diff
@@ -8,8 +8,6 @@ For local development inside a Docker container, start the Docker container and
 ```bash
 cd /localhost/
 cd <path to the repo on localhost>
-# Set the base path to the `stacks` and `components` folders
-export ATMOS_BASE_PATH=$(pwd)
```

Comment:
I think you should leave in:
```
# Set the base path to the `stacks` and `components` folders
export ATMOS_BASE_PATH=$(pwd)
```
because Geodesic only sets `ATMOS_BASE_PATH` when you launch it, and then only based on your current working directory. If you launch Geodesic from your home directory and then CD to your repo directory, you will still need to set `ATMOS_BASE_PATH` yourself.